app/views/homepage/index.html.erb

```diff
@@ -34,30 +34,7 @@
   </header>
 
   <div id="homepage" class="govuk-width-container">
-    <div class="homepage__ready-container govuk-grid-row">
-      <h2 class="homepage__ready-heading govuk-heading-m"><%= I18n.t("homepage.get_ready") %></h2>
-
-      <div class="govuk-grid-column-two-thirds govuk-!-padding-0">
-        <p class="govuk-body">A Brexit deal has been agreed in principle with the EU.</p>
-        <p class="govuk-body">Both the UK and the EU need to approve and sign the withdrawal agreement.</p>
-        <p class="govuk-body homepage__ready-call-to-action">
-          <%= link_to "Find out what this means for you",
-            "/brexit",
-            class: "govuk-link",
-            data: {
-              "module": "track-click",
-              "track-category": "brexit-homepage",
-              "track-label": "/brexit",
-              "track-action": "Call to action link"
-            }
-          %>
-        </p>
-      </div>
-    </div>
-
-    <hr class="home-divider" aria-hidden="true" />
-
-    <section class="home-services" aria-labelledby="services-and-information-label" id="services-and-information">
+    <section class="home-services homepage__ready-container" aria-labelledby="services-and-information-label" id="services-and-information">
```

Comment:
I'd suggest not using the `homepage__ready-container` class here, as it's a bit confusing (was previously used for the chevron section, and now we're moving it to something which isn't "getting ready for Brexit"). I'd suggest tweaking the `home-services` class padding instead until it matches what we want.

Reply:
amended